tools/mo/requirements_caffe.txt

```diff
@@ -1,5 +1,5 @@
 -c ../constraints.txt
-numpy>=1.16.6,<1.26
+numpy>=1.16.6,<1.27
```

Comment:
I suppose it should be either `<2.0` or `<=1.26.4` as version 1.27 does not exist and referring to it is slightly confusing.

Reply:
I'd prefer we avoid `<2.0` in case `1.27` shows up and contains a breaking change (has happened in the past). Moreover, `<1.27` is superior over `<=1.26.4` because if any patch releases show up we'll still get them. With `<=1.26.4` we would miss a potential `numpy==1.26.5`.